src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -81,6 +84,10 @@ protected override void ProcessRecord()
                     {
                         ProcessMarkdownInfo(markdownInfo);
                     }
+                    else if (InputObject.BaseObject is string objectString)
+                    {
+                        _inputObjectBuffer.Add(objectString);
+                    }
```

Comment:
I guess we could get weird result if we get not only strings from pipeline.

Reply:
Yeah that was the main risk I saw with this approach. Tbh though even before you could pass in mixed results, e.g. `markdowninfo`, string and another type together and I'm not even sure what behaviour you'd get. There is an `else` which basically processes any type.

Reply:
Although currently its all done in `ProcessRecord` together, so with this new behaviour we could get some weird order since strings are all processed in `EndProcessing` now. We may need some validation perhaps to now allow this?

Reply:
Perhaps we should think about emulating `Get-WhatsNew | Out-String | Show-Markdown` - convert all inputs to strings and create final string.

Reply:
I do think it's worthwhile, too many different types being checked, it would be better to consolidate all to just `string` and then we have an easier time and less risk of strange results 😄.